src/main/resources/blockly/toolbox_ai_builder.xml

```diff
@@ -0,0 +1,8 @@
+<xml id="toolbox">
+    <custom-other/>
+    <category name="${t:blockly.category.advanced}" colour="300">

```

Comment:
I would set this color to 250 so it matches advanced from procedures, and change maybe "other tasks" category and its procedure blocks to this color (300)

Reply:
I think it would keep things more uniform / unified